docs/aws.md

```diff
@@ -2,322 +2,380 @@
+## Build the Cluster
 
-#### Apply the changes
+Now we take the final step of actually building the cluster.  This'll take a
+while.  Once it finishes you'll have to wait longer while the booted instances
+finish downloading Kubernetes components and reach a "ready" state.
 
 ```bash
 kops update cluster ${NAME} --yes
 ```
 
+## Use the Cluster
 
-## Accessing your cluster
-
-A friendly reminder that kops runs asynchronously, and it will take your cluster a few minutes to come online.
-
-Remember when you installed `kubectl` earlier? The configuration for your cluster was automatically generated and written to `~/.kube/config` for you!
+Remember when you installed `kubectl` earlier? The configuration for your
+cluster was automatically generated and written to `~/.kube/config` for you!
 
-A simple Kubernetes API call can be used to check if the API is online and listening. Let's use `kubectl`
+A simple Kubernetes API call can be used to check if the API is online and
+listening. Let's use `kubectl` to check the nodes.
 
 ```bash
 kubectl get nodes
 ```
 
-You will see a list of nodes that should match the `--zones` flag defined earlier. This is a great sign that your Kubernetes cluster is online and working.
+You will see a list of nodes that should match the `--zones` flag defined
+earlier. This is a great sign that your Kubernetes cluster is online and
+working.
 
-Also kops ships with a handy validation tool that can be ran to ensure your cluster is working as expected.
+Also `kops` ships with a handy validation tool that can be ran to ensure your
+cluster is working as expected.
 
 ```bash
 kops validate cluster
 ```
 
-Another great one liner
+You can look at all the system components with the following command.
 
 ```
 kubectl -n kube-system get po
 ```
 
-## What's next?
+# What's next?
 
-Kops has a ton of great features, and an amazing support team. We recommend researching [other interesting modes](commands.md#other-interesting-modes) to learn more about generating Terraform configurations, or running your cluster in HA (Highly Available). You might want to take a peek at the [cluster spec docs](cluster_spec.md) for helping to configure these "other interesting modes". Also be sure to check out how to run a [private network topology](topology.md) in AWS.
+We've barely scratched the surface of the capabilities of `kops` in this guide,
+and we recommend researching [other interesting
+modes](commands.md#other-interesting-modes) to learn more about generating
+Terraform configurations, or running your cluster in an HA (Highly Available)
+mode.
 
-
-
-Explore the program, and work on getting your `cluster config` hammered out!
+The [cluster spec docs](cluster_spec.md) can help to configure these "other
```

Comment:
This is really great,

Will need to validate these docs as well.. and consider how we want to handle versioning out API docs https://github.com/kubernetes/kops/issues/1583 in the future